Tests/WriteFreelyTests/WriteFreelyClientCollectionTests.swift

```diff
@@ -0,0 +1,155 @@
+import XCTest
+@testable import WriteFreely
+
+final class WriteFreelyClientCollectionTests: XCTestCase {
+    static var allTests = [
+        ("testCreateCollection_WithValidCollectionData_CreatesNewWFCollection", testCreateCollection_WithValidCollectionData_CreatesNewWFCollection),
+        ("testCreateCollection_WithInvalidCollectionData_ReturnsBadRequestError", testCreateCollection_WithInvalidCollectionData_ReturnsBadRequestError),
+        ("testGetCollection_WithValidCollectionData_RetrievesCollectionMetadata", testGetCollection_WithValidCollectionData_RetrievesCollectionMetadata),
+        ("testGetCollection_WithInvalidCollectionData_ReturnsInvalidDataError", testGetCollection_WithInvalidCollectionData_ReturnsInvalidDataError),
+        ("testDeleteCollection_WithValidCollectionAlias_ReturnsTrue", testDeleteCollection_WithValidCollectionAlias_ReturnsTrue),
+        ("testDeleteCollection_WithInvalidCollectionAlias_ReturnsInvalidResponseError", testDeleteCollection_WithInvalidCollectionAlias_ReturnsInvalidResponseError)
+    ]
```

Comment:
We focus here on the `createCollection`, `getCollection`, and `deleteCollection` methods on WFClient as "collection"-related methods.

The REST API includes things like getting a post from a collection, for example, but in the client we make that a "post"-related method as an overload on `getPost`.